### PR TITLE
Add a Java Thai book

### PR DESCRIPTION
**Pull request by tumit**
## What does this PR do?
Add resource(s) 

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-th.md [hunk 1]
```diff
@@ -32,9 +32,13 @@
...
 * [Java Programming Concept](http://it.e-tech.ac.th/poohdevil/JavaConcepts/) - Rungrote Phonkam
 
-
 ### Python
 
 * [Python ๑๐๑](https://www.cp.eng.chula.ac.th/books/python101/) - กิตติภณ พละการ, กิตติภพ พละการ, สมชาย ประสิทธิ์จูตระกูล, สุกรี สินธุภิญโญ
+* [เชี่ยวชาญการเขียนโปรแกรมด้วยไพธอน Programming expert with Python](https://www.theconfig.me/p/programming-expert-with-python.html) - ผศ.สุชาติ คุ้มมะณี
+
+### Unix
+* [พื้นฐานระบบปฏิบัติการยูนิกซ์ (Fundamentals of UNIX)](https://drive.google.com/drive/folders/1gsKVFQHNHDUTCDvQ-ABE3xCHt3nGvfFV?fbclid=IwAR2VZKqFPWxqpuYI2lMVWibLdLFAT4Ee-ldLfDpH7vY-RWBrV6mLpJaMf10) - ผศ.สุชาติ คุ้มมะณี
```
Comment (eshellman) [inline books/free-programming-books-th.md:44]: In general, we do not link to file sharing sites, or file hosting sites such as google docs. or dropbox.

File: books/free-programming-books-th.md [hunk 2]
```diff
@@ -32,9 +32,13 @@
...
 * [โครงสร้างข้อมูลฉบับวาจาจาวา](https://www.cp.eng.chula.ac.th/books/ds-vjjv/) - สมชาย ประสิทธิ์จูตระกูล
 * [แบบฝึกปฏิบัติฉบับวาจาจาวา](https://www.cp.eng.chula.ac.th/books/java101-lb/) - สมชาย ประสิทธิ์จูตระกูล
+* [เริ่มเรียนเขียนโปรแกรม: ฉบับวาจาจาวา](https://www.cp.eng.chula.ac.th/books/bp-vjjv/) - สมชาย ประสิทธิ์จูตระกูล
 * [Java Programming Concept](http://it.e-tech.ac.th/poohdevil/JavaConcepts/) - Rungrote Phonkam
 
-
 ### Python
 
 * [Python ๑๐๑](https://www.cp.eng.chula.ac.th/books/python101/) - กิตติภณ พละการ, กิตติภพ พละการ, สมชาย ประสิทธิ์จูตระกูล, สุกรี สินธุภิญโญ
+* [เชี่ยวชาญการเขียนโปรแกรมด้วยไพธอน Programming expert with Python](https://www.theconfig.me/p/programming-expert-with-python.html) - ผศ.สุชาติ คุ้มมะณี
```
Comment (eshellman) [inline books/free-programming-books-th.md:41]: Can't tell if this is an authorized distribution. The website does not appear to be relate to the author

**Review by eshellman — CHANGES_REQUESTED** [review]
first entry is fine, I suggest removing the others, and if research determines they are authorized by the creators, make another PR.

File: books/free-programming-books-th.md (hunk 1 above)
Comment (tumit) [reply books/free-programming-books-th.md]: OK

File: books/free-programming-books-th.md (hunk 2 above)
Comment (tumit) [reply books/free-programming-books-th.md]: OK

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**